### PR TITLE
Azure Batch REEF command builder

### PR DESCRIPTION
**Pull request by homezcx**

### CODE REVIEW

File: lang/cs/Org.Apache.REEF.Client/AzureBatch/Util/AbstractCommandBuilder.cs [hunk 1]
```diff
@@ -16,22 +16,56 @@
...
 using System.Text;
-using System.Threading.Tasks;
+using Org.Apache.REEF.Common.Files;
 
 namespace Org.Apache.REEF.Client.AzureBatch.Util
 {
     internal abstract class AbstractCommandBuilder : ICommandBuilder
     {
-        public string BuildDriverCommand()
+        private const string JAVA_EXE = @"java";
```
Comment (ryumbra) [inline lang/cs/Org.Apache.REEF.Client/AzureBatch/Util/AbstractCommandBuilder.cs]: I believe that in C# the standard way to name constants is camel-case with first letter uppercase, ie. JavaExe. I took a quick look at how these variables are named elsewhere and REEF.NET seems to be following that convention.

File: lang/cs/Org.Apache.REEF.Client/AzureBatch/Util/AzureBatchFileNames.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+namespace Org.Apache.REEF.Client.AzureBatch.Util
+{
+    /// <summary>
+    /// Access to the various places things go according to the REEF Azure Batch runtime.
+    /// </summary>
+    internal sealed class AzureBatchFileNames
+    {
+        private const string STORAGE_JOB_FOLDER_PATH = "apps/reef/jobs/";
+        private const string TASL_JAS_FILE_NAME = "local.jar";
```
Comment (ryumbra) [inline lang/cs/Org.Apache.REEF.Client/AzureBatch/Util/AzureBatchFileNames.cs]: Same as above; also TASK_JAR_FILE_NAME :)

**Review by ryumbra — APPROVED** [review]
Looks good. The only thing is the constant naming convention.

File: lang/cs/Org.Apache.REEF.Client/AzureBatch/Util/AbstractCommandBuilder.cs (hunk 1 above)
Comment (homezcx) [reply lang/cs/Org.Apache.REEF.Client/AzureBatch/Util/AbstractCommandBuilder.cs]: Fixed. But REEFFileNames.cs is using this format.

File: lang/cs/Org.Apache.REEF.Client/AzureBatch/Util/AzureBatchFileNames.cs (hunk 2 above)
Comment (homezcx) [reply lang/cs/Org.Apache.REEF.Client/AzureBatch/Util/AzureBatchFileNames.cs]: fixed.

**Pull request merged**